init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/model/PractitionerUserAccount.java

```diff
@@ -0,0 +1,28 @@
+package uk.ac.ox.ndph.mts.init_service.model;
+
+public class PractitionerUserAccount implements Entity {
+    private String practitionerId;
+    private String userAccountId;
+
+    public String getUserAccountId() {
+        return userAccountId;
+    }
+
+    public String getPractitionerId() {
+        return practitionerId;
+    }
+
+    public void setPractitionerId(String practitionerId) {
+        this.practitionerId = practitionerId;
+    }
+
+    public void setUserAccountId(String directoryId) {
```

Comment:
Not an important point but should  the directoryId arg be called accountId?  (In practice this is an AD ID of course.)